model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/trivy/TrivyDto.kt

```diff
@@ -29,7 +29,10 @@ data class TrivyDtoV2(
 
 @Serializable
 data class Result(
-    @SerialName("Vulnerabilities") val vulnerabilities: List<TrivyVulnerabilityDto> = listOf()
+    @SerialName("Vulnerabilities") val vulnerabilities: List<TrivyVulnerabilityDto> = listOf(),
```

Comment:
could you please quickly check whether we use default values in our other data classes? If I remember correctly, we opted to not use default values at all and leave it to the library's users to decide how to do the initialization, but I'm not 100% from the top of my head :)

Reply:
Defaults values are used in the tlc's [project dto](https://github.com/fraunhofer-iem/spha/blob/main/model/src/main/kotlin/de/fraunhofer/iem/spha/model/adapter/tlc/ProjectDto.kt). The other dtos does not have default values. If we're going to make all dtos not have default values, I think its best to make another PR, since tests needs to be changed too.

Reply:
agreed. I think it would be good if we also change the tlc's dtos. So let's merge this and do a dedicated PR  as you suggested